tests/publiccloud/run_ltp.pm

```diff
@@ -94,22 +90,27 @@ sub run {
     $self->{ltp_env} = $ltp_env;
 
     # Use lib/LTP/WhiteList module to exclude tests
-    if (get_var('LTP_KNOWN_ISSUES')) {
-        my $whitelist = LTP::WhiteList->new();
+    my $issues = get_var('LTP_KNOWN_ISSUES', '');
+    if ($issues) {
+        my $whitelist = LTP::WhiteList->new($issues);
         my $exclude = get_var('LTP_COMMAND_EXCLUDE', '');
-        my @skipped_tests = $whitelist->list_skipped_tests($ltp_env, get_required_var('LTP_COMMAND_FILE'));
-        if (@skipped_tests) {
-            $exclude .= '|' if (length($exclude) > 0);
-            $exclude .= '^(' . join('|', @skipped_tests) . ')$';
-            set_var('LTP_COMMAND_EXCLUDE', $exclude);
```

Comment:
This part is also needed because we are rebuilding from scratch that LTP_COMMAND_EXCLUDE parmeter, based on `@skipped`. Here I see missing as the LTP_COMMAND_EXCLUDE assignment as the needed enveloping brackets `^(...)$`: isn't it?

Reply:
supper seeded `$skip_test` and opted to not modify test variable --> it merged result of `LTP_KNOWN_ISSUES` and `LTP_COMMAND_EXCLUDE` and passes it to runner parameter

Reply:
the lexical syntax of ^( ... )$ is an _inversion_ of logic you cannot remove it.

Reply:
> the lexical syntax of ^( ... )$ is an _inversion_ of logic you cannot remove it.

no , its **regexp** so it only constraint for match only whole strings in this constraint ... btw not even `LTP::kirk` cares about this and simply constructs `$skip_tests = 'test1|test2|test_to_skip`

and we can use same :D

but we can wrap result of LTP_KNOWN_PROBLEMS in this constraints, it should not have any adverse effect on run